cmd/celestia/rpc.go

```diff
@@ -402,12 +402,13 @@ func rpcClient(ctx context.Context) (*client.Client, error) {
 }
 
 func printOutput(data interface{}, err error, formatData func(interface{}) interface{}) {
-	if err != nil {
-		data = err
-	} else {
+	switch err {
+	case nil:
 		if formatData != nil {
 			data = formatData(data)
 		}
+	default:
+		data = err
 	}
```

Comment:
should just be a switch where
 
case err != nil : 
 data = formatData(err)
case formatData != nil: 
  data = formatData(data)
default: 
 data = formatData(fmt.Errorf(" problem with output"))

Reply:
format data could be nil. And we don't need to format errors. But I got your idea